packages/cfd/src/Components/cfd-mt5-demo-account-display.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { CFDAccountCard } from './cfd-account-card';
+import { general_messages } from '../Constants/cfd-shared-strings';
+import specifications, { TSpecifications } from '../Constants/cfd-specifications';
+import Loading from '../templates/_common/components/loading';
+import { DetailsOfEachMT5Loginid, LandingCompany } from '@deriv/api-types';
+import { TTradingPlatformAccounts } from './props.types';
+import { TObjectCFDAccount } from 'Containers/cfd-dashboard';
+
+type TStandPoint = {
+    financial_company: string;
+    gaming_company: string;
+    iom: boolean;
+    malta: boolean;
+    maltainvest: boolean;
+    svg: boolean;
+};
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+type TCFDDemoAccountDisplayProps = {
+    is_eu: boolean;
+    is_eu_country: boolean;
+    has_maltainvest_account: boolean;
+    has_cfd_account_error: boolean;
+    openAccountNeededModal: (target: string, target_label: string, target_dmt5_label: string) => void;
+    standpoint: TStandPoint;
+    is_loading: boolean;
+    is_logged_in: boolean;
+    isSyntheticCardVisible: (account_category: string) => boolean;
+    isFinancialCardVisible: () => boolean;
+    onSelectAccount: (objCFDAccount: TObjectCFDAccount) => void;
+    openAccountTransfer: (
+        data: DetailsOfEachMT5Loginid | TTradingPlatformAccounts,
+        meta: TOpenAccountTransferMeta
+    ) => void;
+    platform: string;
```

Comment:
```suggestion
    platform: keyof TSpecifications;
```
And we can remove all the unsafe type castings.

Reply:
I use `TCFDPlatform` which is the type for platform types.